app/src/main/res/layout/fragment_add_bookmark_folder.xml

```diff
@@ -3,11 +3,12 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:layout_margin="16dp"
-    android:orientation="vertical">
+        xmlns:tools="http://schemas.android.com/tools"
```

Comment:
Is there any particular reason why we want to change the indentation? :)

Reply:
Android Studio automatically uses this indentation, which is probably the cause here.